compiler-rt/lib/nsan/nsan_flags.inc

```diff
@@ -47,3 +47,5 @@ NSAN_FLAG(bool, enable_loadtracking_stats, false,
           "memory, the number of times nsan resumed from the original value "
           "due to invalid or unknown types.")
 NSAN_FLAG(bool, print_stats_on_exit, false, "If true, print stats on exit.")
+NSAN_FLAG(bool, check_nan, true,
```

Comment:
i remember the comment about making `check_nan` `true`,  I'm a bit worried that this check might fire early in places where nans are legitimate... Not sure, but intuitively I'd probably keep it "false" by default, at least for now.

Reply:
Sure, I will change this option.